index.php

```diff
@@ -92,7 +92,7 @@
             <div class="app-display app-display--overflow">
                 <img class="app-display__image" src="images/screenshots/appcenter.png" srcset="images/screenshots/appcenter@2x.png 2x" alt="elementary OS AppCenter home page"/>
                 <div class="app-display__description">
-                    <img src="images/icons/apps/128/system-software-install.svg" />
+                    <img src="images/icons/apps/128/system-software-install.svg" alt="elementary OS AppCenter icon"/>
```

Comment:
When disambiguating application names, the convention is "elementary Foo" rather than "elementary OS Foo" 
```suggestion
                    <img src="images/icons/apps/128/system-software-install.svg" alt="elementary AppCenter icon"/>
```